utils/heartbeat.js

```diff
@@ -0,0 +1,28 @@
+const { logIPAdddress, setHeaders } = require('./shared');
+
+const heartbeat = async (req, res) => {
+    logIPAdddress(req);
```

Comment:
small typo I think

Reply:
unfortunately that's the "right" spelling, but I'll correct it